client/gutenberg/editor/controller.js

```diff
@@ -102,6 +106,11 @@ export const loadGutenbergBlockAvailability = store => {
 	} );
 };
 
+export const resetGutenbergState = registry => {
+	registry.reset();
+	return {};
+};
```

Comment:
This is a plugin that's registered repeatedly on every route load, and all it does is to call the `registry.reset()` method? We do it this way because there is no other way to access the `registry` object and call a method from it? The `@wordpress/data` package exports only few selected methods of the registry.